app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,73 @@
+
 .check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+  > * {
```

Comment:
Would be a good idea to apply classes rather than wildcards, this way people don't have to put aggressive overrides if they want to extend this pattern.